flow-typed/npm/winston_v3.x.x.js

```diff
@@ -81,7 +81,9 @@ declare type $winstonLoggerConfig<T: $winstonLevels> = {
 };
 
 declare type $winstonLogger<T: $winstonLevels> = {
-    [$Keys<T>]: (message: string, meta?: Object) => void,
+    // Start: https://github.com/Khan/render-gateway/pull/??
+    [$Keys<T>]: (message: string, metaOrCallback?: Object | () => void, callback?: () => void) => void;
+    // End: https://github.com/Khan/render-gateway/pull/??
```

Comment:
I will update the comments with this PR number, now it has one.

This means we properly support the callback method that ensures all transports have received a logged message.